### DISCUSSION TITLE
blog/devlog-4-buildings/

### DISCUSSION THREAD

**Discussion by giscus** [modulocity/automationstationgame.com#6] [Comments]
# blog/devlog-4-buildings/

Hey folks!

https://automationstationgame.com/blog/devlog-4-buildings/

**Comment by alijaya** [comment]
Just found out this amazing blog! It's really great that you share your design process.

For me, I'm more in the side of Crafted Building in Factorio, never a fan of Satisfactory process.

I think it's depend on the game that you want to make... In Satisfactory, you usually only care about the item, or crafting intermediate resource. Because the resource sink only asks about intermediate resource.

For the building, you just need the required resource, and you could build it freely.
This type of building happens in Terraforming Mars also, and Oddsparks.

The downside of this one is, you need to calculate how much resource that you need to have in your inventory to create for example 5 building. And of course, your inventory need much more larger, because you bring all the resource instead of actual building.

In Astroneer, with limited inventory (like yours), they opt to Crafted Building, because they can't take much item with them.

But well... I like the approach with Factorio, because in Factorio, expansion is one of the priority, and making building is one of the resource sink, and also a puzzle in itself. To make it auto constructed from raw resource means removing some of the puzzle.
